shell/browser/notifications/mac/cocoa_notification.h

```diff
@@ -36,7 +35,7 @@ class CocoaNotification : public Notification {
  private:
   void LogAction(const char* action);
 
-  base::scoped_nsobject<NSUserNotification> notification_;
+  NSUserNotification* notification_;
```

Comment:
This needs a `__strong` per Chromium conventions:

> Chromium usage note: Even though this is the default, please always explicitly specify it for class members/ivars so that it is clear which pointers are Objective-C object pointers and which are C++ object pointers.

Reply:
Thanks! (And thanks for the Chromium note too, this is really helpful) 😄 Fixed this and others below